sim/src/lib.rs

```diff
@@ -11,6 +11,9 @@ mod sim;
 mod transit;
 mod trips;
 
+#[macro_use]
+extern crate log;
```

Comment:
Generally I like to have macros namespaced, but logging is one of those things that I want to use everywhere without having to `use log` on every file.

But if it's controversial I can revert it.